### ISSUE TITLE
In certain cases, ignoring props doesn't work

### ISSUE THREAD

**Issue by lancetharper** [vue-styleguidist/vue-styleguidist#6]
Following [the docs](https://github.com/vue-styleguidist/vue-styleguidist/blob/master/docs/Documenting.md#ignoring-props).

I tried listing ignored props first, as seen in the example, and at the end of the props object but they are not ignored in the docs either way.

**Comment by rafaesc92** [comment]
It's true, I'll update it

**Comment by lancetharper** [comment]
It looks like you can't have multiple ignores. If you try to, only the last one is actually ignored.

**Comment by rafaesc92** [comment]
Yes, when I finish improving vue-docgen-api, I will integrate the ignore props solution into vue-styleguidist. In this repository, I need to add the filter

**Comment by rafaesc92** [comment]
The repository was updated.

Upgrade to version 1.1.5

**Comment by lancetharper** [comment]
Awesome, thanks!

Not a bug per se, but I noticed this doesn't ignore props that are defined with the prop shorthand.

**not** ignored:
```
/** @ignore */
foo: String,
```

**is** ignored:
```
/** @ignore */
foo: {
    type: String,
}
```

**Comment by rafaesc92** [comment]
I will solve the problem, by the way, add a new example of vue-styleguidist, I hope this can help you with your project.

https://github.com/vue-styleguidist/buefy-styleguide-example

**Comment by rafaesc92** [comment]
Fixed issue, please upgrade to version 1.1.6
